packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -131,6 +103,10 @@ class _TimelineFlameChartState
         curve: defaultCurve,
       );
 
+      // Maybe bail early if multiple frame selections were triggered in
```

Comment:
Nit for this comment. Make it more perscriptive.
E.g.
` // Bail early if the selection has changed again while the animation was in progress.`